pontoon/settings/base.py

```diff
@@ -873,9 +873,9 @@ def _default_from_email():
 # Use correct header for detecting HTTPS on Heroku.
 SECURE_PROXY_SSL_HEADER = ("HTTP_X_FORWARDED_PROTO", "https")
 
-# Do not set SECURE_HSTS_SECONDS.
-# HSTS is being taken care of in pontoon/wsgi.py.
-# SECURE_HSTS_SECONDS = 63072000
+# If set to a non-zero integer value, the SecurityMiddleware sets the
+# HTTP Strict Transport Security header on all responses that do not already have it.
+SECURE_HSTS_SECONDS = 31536000
```

Comment:
Can we put in the comment that this is 365 days?

Reply:
Updated.